presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -124,6 +124,7 @@
     public static final String QUERY_MAX_TOTAL_MEMORY_PER_NODE = "query_max_total_memory_per_node";
     public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_ROW_COUNT = "dynamic_filtering_max_per_driver_row_count";
     public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_SIZE = "dynamic_filtering_max_per_driver_size";
+    public static final String USE_PARENT_PREFERRED_PROPERTIES = "use_parent_preferred_properties";
```

Comment:
I'm thinking we want a name like "greedy-property-matching" or "greedy-property-enforcement" (as in, consider only property requirements from immediate operation regardless of whether it might not benefit other operations). Or maybe just "ignore-downstream-preferences".